src/main/java/io/antmedia/rest/RequestWrapper.java

```diff
@@ -0,0 +1,81 @@
+package io.antmedia.rest;
+
+import java.io.BufferedReader;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+import javax.servlet.ReadListener;
+import javax.servlet.ServletInputStream;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletRequestWrapper;
+
+import org.apache.commons.io.IOUtils;
+
+public class RequestWrapper extends HttpServletRequestWrapper{
```

Comment:
Yes, otherwise, we need to transmit REST API data in the URL instead of the request body, which is considered a less favorable practice. We already have an ongoing issue aimed at improving our REST API.